arm-network/2016-12-01/examples/RouteFilterCreate.json

```diff
@@ -0,0 +1,99 @@
+{
+  "parameters": {
+    "routeFilterName": "filterName",
+    "resourceGroupName": "rg1",
+    "api-version": "2016-12-01",
+    "subscriptionId": "subid",
+    "parameters": {
```

Comment:
name of the body parameter in the swagger spec is `"routeFilterParameters"` whereas the example contains an incorrect parameter name.

```
error: validating the request for x-ms-example "RouteFilterCreate" in operation "RouteFilters_CreateOrUpdate":

{ code: 'REQUEST_VALIDATION_ERROR',
  message: 'Found errors in validating the request for x-ms-example "RouteFilterCreate" in operation "RouteFilters_CreateOrUpdate".',
  innerErrors:
   [ { code: 'REQUIRED_PARAMETER_EXAMPLE_NOT_FOUND',
       message: 'In operation "RouteFilters_CreateOrUpdate", parameter routeFilterParameters is required in the swagger spec but is not present in the provided example parameter values.' } ] }
```

Please make sure that all the examples contain valid parameter names.